File: src/test/resources/output/factorial.cs
// Código generado desde EBDA
using System;

class Program {
    static void Main(string[] args) {
        int n = 5;
        int resultado = 1;
        int i = 1;
        bool corriendo = true;
        while (corriendo) {
            resultado = resultado * i;
            i++;
            corriendo = i < n;
        }
        resultado = resultado * n;
        Console.WriteLine("Factorial de 5:");
        Console.WriteLine(resultado);
        return;
    }
}

File: src/test/resources/output/factorial.js
// Código generado desde EBDA

let n = 5;
let resultado = 1;
let i = 1;
let corriendo = true;
while (corriendo) {
    resultado = resultado * i;
    i++;
    corriendo = i < n;
}
resultado = resultado * n;
console.log("Factorial de 5:");
console.log(resultado);
return resultado;
